cli.go

```diff
@@ -402,6 +402,10 @@ func (c *Cli) SaveData(data interface{}) error {
 
 func (c *Cli) ViewIssue(issue string) (interface{}, error) {
 	uri := fmt.Sprintf("%s/rest/api/2/issue/%s", c.endpoint, issue)
+	if x := c.expansions(); len(x) > 0 {
```

Comment:
This would be more efficient with ```fmt.Sprintf``` only called once, and the right one being selected per branch of ```if... else...``` but I found it easier to read this way - especially for an unsolicited PR. And even though ```fmt.Sprintf``` isn't super cheap, this tool is not really gonna be impacted by an extra operation or two for most users :snail: